.github/workflows/main.yml

```diff
@@ -23,6 +23,8 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - run: npm ci
+      - run: npm run lint:fix

```

Comment:
Should not fix lint errors on CI, else the errors being fixed will be committed on main repo and propagate to all other users who pull upstream